.github/workflows/argus-stack-staging-upsert.yaml

```diff
@@ -0,0 +1,35 @@
+name: Update staging
+
+on:
+  pull_request:
+    types: [ opened, synchronize ]
+    paths:
+      - '.infra/common.yaml'
+      - '.infra/staging/values.yaml'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  run_if:
+    if: github.head_ref == 'release-please--branches--main--components--frontend'
```

Comment:
I think you'll want to go through and make sure anything referencing `release-please--branches--...` is set to `release-please--branches--main--components--cryoet-data-portal-frontend` now that the release PR was created